Bottom: 80ad50da808856e87b3c8cba7736df9b9791382d
Top:    4ca8527742d2484c76658994103628221448aae6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-19 16:59:05 +0100

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index 32036d7..4f24674 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -188,7 +188,7 @@ DrawStrippedText (ElementTypePtr Element, int min_width)
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   if (doing_pinout || doing_assy)
